### PR TITLE
cleaner implementation of JOB_ROUTER_TRANSFORM_BatchRuntime

### PR DESCRIPTION
**Pull request by johnkn**

### CODE REVIEW

File: config/01-ce-router-defaults.conf.in [hunk 1]
```diff
@@ -258,16 +258,15 @@ JOB_ROUTER_TRANSFORM_BatchRuntime @=jrt
     # 3. 'default_maxWallTime' (minutes) from the route
     # 4. 'ROUTED_JOB_MAX_TIME' (minutes) from the config
     if defined MY.maxWallTime
-        def_walltime = 60*$(MY.maxWallTime)
+        SET BatchRuntime 60*maxWallTime
```
Comment (brianhlin) [inline config/01-ce-router-defaults.conf.in:261]: Should this be the following instead to be more precise?

```suggestion
        SET BatchRuntime 60*MY.maxWallTime
```

Comment (johnkn) [reply config/01-ce-router-defaults.conf.in:261]: Doesn't hurt, but there's no particular reason to use MY there.  it just clutters up the final job expression.

Comment (brianhlin) [reply config/01-ce-router-defaults.conf.in:261]: Ah yeah I suppose that's true since this isn't used for HTCondor batch systems

**Review by brianhlin — APPROVED** [review]
LGTM, osg test failures are unrelated

**Pull request merged**